lib/portlayer/storage/vsphere/toolbox_datasink.go

```diff
@@ -54,10 +56,22 @@ func (t *ToolboxDataSink) Import(op trace.Operation, spec *archive.FilterSpec, d
 
 	// upload the gzip archive.
 	p := soap.DefaultUpload
-	err = client.Upload(op, data, target, p, &types.GuestPosixFileAttributes{}, true)
+	// These numbers are somewhat arbitrary best guesses
+	retryConf := retry.NewBackoffConfig()
```

Comment:
Looks like this is in every file. Can you please make it a constant in toolbox_common?